py/torch_tensorrt/_compile.py

```diff
@@ -502,19 +502,24 @@ def save(
                 "Provided model is a torch.jit.ScriptModule but the output_format specified is exported_program. Please verify the output_format"
             )
         else:
+            if arg_inputs is not None:
+                raise ValueError(
+                    "Provided model is a torch.jit.ScriptModule, do not allow user to provide inputs or arg_inputs."
+                )
             torch.jit.save(module, file_path)
     elif module_type == _ModuleType.ep:
         if output_format == "torchscript":
             raise ValueError(
                 "Provided model is a torch.export.ExportedProgram but the output_format specified is torchscript. Please verify the output_format"
             )
         else:
+            if arg_inputs is not None:
```

Comment:
Same here

Reply:
changed to warning instead of raising error